src/oaklib/cli.py

```diff
@@ -3820,6 +3827,9 @@ def fill_table(
     - TODO: use most optimized method for whichever backend
     """
     tf = TableFiller(settings.impl)
+    with open(table_file) as input_file:
+        comment_lines = [x for x in input_file.readlines() if x.startswith(comment)]
```

Comment:
this is a bit inefficient if it's a large table and we know comments are only at the top